threema/gateway/e2e.py

```diff
@@ -680,7 +683,8 @@ class DeliveryReceipt(Message):
     Each delivery receipt message confirms the receipt of one
     or multiple regular text messages.
 
-    .. note:: Sending delivery receipts is not officially supported.
```

Comment:
I guess it wasn't unsupported, it was just not documented. The E2E message format is now documented at https://gateway.threema.ch/de/developer/api though.